src/Toolkit/Toolkit/LocalizedStrings/Resources.resx

```diff
@@ -117,6 +117,14 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BasemapGalleryAutomationName" xml:space="preserve">
+    <value>Basemap Gallery</value>
+    <comment>UIA AutomationProperties.Name for the BasemapGallery control, used by screen readers to identify the control</comment>
+  </data>
+  <data name="BasemapGalleryAutomationHelpText" xml:space="preserve">
+    <value>Allows selection of different basemap styles for the map</value>
```

Comment:
Small changes to fit our standard terminology:
- Use "basemap" instead of "basemap style" because this control is not limited to enumerated [BasemapStyle](https://developers.arcgis.com/net/api-reference/api/netwin/Esri.ArcGISRuntime/Esri.ArcGISRuntime.Mapping.BasemapStyle.html)s.
- Mention both maps and scenes, since this gallery can be used with either.

I'd also expand this a little bit to fit HelpText's dual role of providing "text alternatives for nontext objects" and "imperative instructions for interactive elements":

```suggestion
    <value>Displays a list of available basemaps. Select a basemap to use it for the current map or scene.</value>
```